demo/Makefile

```diff
@@ -7,9 +7,9 @@ include $(TOP)/build.mk
 
 SUB_DIRS = chaincode client
 
-build test clean clobber:
+build test clean:
 	$(foreach DIR, $(SUB_DIRS), $(MAKE) -C $(DIR) $@ || exit;)
 
-
 clobber:
 	./scripts/teardown.sh --clean-slate
+	$(foreach DIR, $(SUB_DIRS), $(MAKE) -C $(DIR) $@ || exit;)
```

Comment:
I'm not sure I've used clobber before?

Reply:
only you can answer that ;-)   But the goal of this target is to reset the build to a true clean-slate with all build/downloaded artifacts removed.